internal/course/database/course_queries.go

```diff
@@ -1,27 +1,39 @@
 package database
 
+import (
+	"fmt"
+)
+
 const (
-	createCourse       = "create course"
-	deleteCourse       = "delete course by uuid"
-	getCourse          = "get course by uuid"
-	listCourse         = "list course"
-	updateCourseByID   = "update course by uuid"
-	updateCourseByCode = "update course by code"
+	returning_columns = "uuid, code, name, underline, image, image_cover, excerpt, description, created_at, updated_at"
```

Comment:
Wrong code style, rename it to `returningColumns` and add a comment to explain where and for what it is being used.

Reply:
https://github.com/sumelms/microservice-course/pull/19/commits/6244cbf2156cfdfa417826cf5017f6c73799dd89